test/unit/rise-data.html

```diff
@@ -67,7 +67,10 @@
 
       test( "should fire 'rise-data-ping-received'", function( done ) {
         var responseHandler = function( response ) {
-          assert.isTrue( response.detail );
+          assert.property( response.detail, "isCacheRunning" );
+          assert.isTrue( response.detail.isCacheRunning );
+          assert.property( response.detail, "baseCacheUrl" );
+          assert.equal( response.detail.baseCacheUrl, "http://localhost:9494" );
```

Comment:
@rodrigopavezi We force this in the test file - https://goo.gl/hkBM2E , it causes the base cache url to change.